src/interfaces/IMorpho.sol

```diff
@@ -123,6 +123,8 @@ interface IMorphoBase {
     /// `liquidate` from being used under certain market conditions.
     /// - A very high price returned by the oracle can make the computation of `maxBorrow` in `_isHealthy` overflow, or
     /// the computation of `assetsRepaid` in `liquidate` overflow.
+    /// @dev The share price of a market with less than 1e4 assets borrowed can be increased by manipulations, to the
+    /// point where borrowing and repaying overflows.
```

Comment:
Repaying can overflow?

Reply:
yes because it uses totalBorrowShares almost the same way as borrow

Reply:
But how can total borrow shares overflow if it's only decreased upon repay?
I think I'm missing something

Reply:
my bad, I thought that the overflow happened during a WAD mul